frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -40,6 +40,12 @@ end
 function ReaderWikipedia:registerKeyEvents()
     if Device:hasKeyboard() then
         self.key_events.ShowWikipediaLookup = { { "Alt", "W" }, { "Ctrl", "W" } }
+        if Device.model == "Kindle3" then
+            -- Override previous shortcut on kindle3 as the kernel designates Alt+W as "2".
+            -- Any user-facing documentation should _still_ advertise this shortcut as Alt+W, as
+            -- that is what the user must input in order to use it.
+            self.key_events.ShowWikipediaLookup = { { "2" } }
```

Comment:
```suggestion
            self.key_events.ShowWikipediaLookup = { { Device.AltPlusQWER["W"] } }
```

Reply:
I don't like that name AltPlusQWER:
- it mentions QWER, but what about TYUIOP ? Will it work ? Nothing strange using Device.AltPlusQWER["U"] ?
- on some other obscure Kindle never released, it may work with QSDFGHHJ ! so, be generic, assume it could be use for some other mapping
- it's a property, not a class or method, so naming should be lowercase with underscores (I get it reads well as AltPlusXyz).

So, maybe `Device.alt_plus_key_kernel_translated["W"]` ?

Reply:
Let's see what @NiLuJe think of my also ugly name suggestion :)